packages/storybook-a11y-test/src/index.ts

```diff
@@ -96,13 +96,19 @@ const isAutcompleteNope = violation => {
   return isAutocompleteAttribute && hasNope;
 };
 
-const testPage = (iframePath, browser, timeout) => {
+const testPage = (iframePath, browser, timeout, disableAnimation) => {
   return async id => {
     console.log(` - ${id}`);
 
     try {
       const page = await browser.newPage();
       await page.goto(`${iframePath}?id=${id}`, {waitUntil: 'load', timeout});
+      if (disableAnimation) {
+        await page.addStyleTag({
+          content:
+            '* {transition: none !important; animation: none !important; }',
+        });
```

Comment:
Is it possible that it could break scripts that rely on `animationend`?

If that's fine, then let's make it more aggressive, and also target pseudo elements ([source](https://github.com/zaqqaz/visual-unit-tests/blob/e4ebade1b8e046dbdbf659983b987ae21566221f/src/utils/testUtils.ts#L61-L70)):

```css
*,
*::after,
*::before {
  transition-delay: 0s !important;
  transition-duration: 0s !important;
  animation-delay: -0.0001s !important;
  animation-duration: 0s !important;
  animation-play-state: paused !important;
  caret-color: transparent !important;
}
```

However, if that's going to be an issue, I suggest using this hack to accelerate animations and might work as well:

```js
await page._client.send('Animation.setPlaybackRate', {playbackRate: 10});
```

Reply:
I tried disable with the client animation options but couldn't get them to work as I expected them too. I updated the styles to include pseudo elements and instead of pausing and changing the delay, I changed all of the durations to a millisecond to make sure the `animationend` events still fire. I also was unsure about why the caret color was needed there, so I didn't include that part.